src/PreferencesController.ts

```diff
@@ -14,19 +16,54 @@ const { toChecksumAddress } = require('ethereumjs-util');
  * @property lostIdentities - Map of lost addresses to ContactEntry objects
  * @property selectedAddress - Current coinbase account
  * @property tokens - List of tokens associated with the active vault
+ * @property collectibles - List of collectibles associated with the active vault
  */
 export interface PreferencesState extends BaseState {
+	collectibles: Collectible[];
 	featureFlags: { [feature: string]: boolean };
 	identities: { [address: string]: ContactEntry };
 	lostIdentities: { [address: string]: ContactEntry };
 	selectedAddress: string;
 	tokens: Token[];
 }
 
+/**
+ * @type Collectible
```

Comment:
Could we move the `Collectible` and `CollectibleCustomInformation` type definitions above the `State` and `Config` interfaces? For consistency, I've been keeping custom types and interfaces above `State` and `Config` interfaces in each controller.